snapraid-runner.conf.example

```diff
@@ -7,6 +7,8 @@ config = snapraid.conf
 deletethreshold = 40
 ; if you want touch to be ran each time
 touch = false
+; prints a SMART report of all the disks of the array.
+smart = true
```

Comment:
I'd prefer for this to be false. Otherwise the script will fail per default if the user doesn't have smartctrl. Also a note `Make sure smartctl is in your PATH.` in the comment would be helpful.